content/en/docs/concepts/storage/volumes.md

```diff
@@ -119,10 +119,10 @@ beta features must be enabled.
 
 #### AWS EBS CSI migration complete
 
-{{< feature-state for_k8s_version="v1.17" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
```

Comment:
Can we separate out this 1.21 feature gate update from the volume deprecations since the latter is being targeted for 1.22 and hasn't been released yet?

The PR for volume deprecations should be targeted to the dev-1.22 branch.